tex/main.run.xml

```diff
@@ -1,85 +0,0 @@
-<?xml version="1.0" standalone="yes"?>
```

Comment:
`git rm tex/main.run.xml`

It won't wipe the history but it will get rid of it.